druid/src/widget/svg.rs

```diff
@@ -85,12 +85,19 @@ impl<T: Data> Widget<T> for Svg<T> {
     }
     fn paint(&mut self, paint_ctx: &mut PaintCtx, _data: &T, _env: &Env) {
         //TODO: options for aspect ratio or scaling based on height
-        let scale = paint_ctx.size().width / self.get_size().width;
+        let scalex = paint_ctx.size().width / self.get_size().width;
+        let scaley = paint_ctx.size().height / self.get_size().height;
+        let scale = scalex.min(scaley);
 
         let origin_x = (paint_ctx.size().width - (self.get_size().width * scale)) / 2.0;
         let origin_y = (paint_ctx.size().height - (self.get_size().height * scale)) / 2.0;
         let origin = Point::new(origin_x, origin_y);
 
+        let clip_rect = Rect::ZERO.with_size(paint_ctx.size());
+
+        // The SvgData's to_piet function dose not clip to the svg's size
```

Comment:
Can get rid of this comment, otherwise looks good to me!

Reply:
I'm happy enough to merge with this here, I think it's a reasonable observation.